testsuite/cluster-test/src/cluster.rs

```diff
@@ -9,11 +9,25 @@ use std::{thread, time::Duration};
 pub struct Cluster {
     // guaranteed non-empty
     instances: Vec<Instance>,
-    prometheus_ip: String,
+    prometheus_ip: Option<String>,
+    mint_file: String,
 }
 
 impl Cluster {
-    pub fn discover(aws: &Aws) -> failure::Result<Self> {
+    pub fn from_host_port(peers: Vec<(String, u32)>, mint_file: &str) -> Self {
+        let instances: Vec<Instance> = peers
+            .into_iter()
+            // WIP(ankush): What should be short_hash here?
+            .map(|host_port| Instance::new("".to_string(), host_port.0, host_port.1))
```

Comment:
Question: What should be short_hash here?

Reply:
So I was thinking that --peers would contain `short_hash:ac_port`
For libra swarm host is always localhost, so we can assume that